package.json

```diff
@@ -39,8 +46,16 @@
     "@babel/preset-env": "^7.10.4",
     "@babel/preset-react": "^7.10.4",
     "@babel/register": "^7.10.5",
+    "babel-eslint": "^10.1.0",
     "babel-loader": "^8.1.0",
+    "babel-preset-airbnb": "^5.0.0",
     "dotenv": "^8.2.0",
+    "eslint": "^7.6.0",
+    "eslint-config-airbnb": "^18.2.0",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jsx-a11y": "^6.3.1",
+    "eslint-plugin-react": "^7.20.5",
+    "eslint-plugin-react-hooks": "^4.0.8",
     "style-loader": "^1.2.1",
     "underscore": "^1.10.2",
```

Comment:
if you use underscrore in your actual code, then likely it will break in production if not included in your dependencies